test/src/pumps_test.dart

```diff
@@ -1,9 +1,41 @@
 import 'package:alchemist/alchemist.dart';
+import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:mocktail/mocktail.dart';
+import 'package:mocktail_image_network/mocktail_image_network.dart';
+
+import '../helpers/helpers.dart';
 
 class MockWidgetTester extends Mock implements WidgetTester {}
 
+Future<bool> _isCached(ImageProvider<Object> image, Finder findImage) async {
+  final cacheStatus = await image.obtainCacheStatus(
+    configuration: createLocalImageConfiguration(
+      findImage.evaluate().first,
+    ),
+  );
+  return cacheStatus?.keepAlive ?? false;
+}
+
+extension on CommonFinders {
```

Comment:
thoughts on moving this to a `helpers/finders.dart` file?

Reply:
I don't think we'll use them outside this file, so I'd rather keep them here.